src/components/navigation/header/index.stories.tsx

```diff
@@ -49,9 +50,17 @@ const meta: Meta<typeof Navigation> = {
   component: Wrapper,
   decorators: [
     (Story) => (
-      <Box fontFamily="Make It Sans" position="relative" height="250px">
-        <Story />
-      </Box>
+      <PageLayout header={<Story />} maxContentWidth="2xl">
```

Comment:
Added page layout here so that we can have similar behaviour as the webapp in storybook and be able to debug better without consuming the packatge